### DISCUSSION TITLE
Welcome to Neverball discussions!

### DISCUSSION THREAD

**Discussion by parasti** [Neverball/neverball#255] [Announcements]
## 👋 Welcome!
  Since the Neverforum has been stuck in limbo for about six months and I just discovered Github discussions, I figured we could check them out.

Feel free to post Neverball things you've made, Neverball things you've achieved, and Neverball questions you've pondered. Neverputt is welcome, too.

**Comment by KodeBreaker-Has-SWAG** [comment]
I never noticed GitHub discussions. This is a great way to discuss about Neverball!

**Comment by Adacrop-The-Great** [comment]
This is much better than the old Neverforum! An excellent way to stay up to date with the latest releases!

**Comment by reshaun** [comment]
Oh! I'm at Neverball GitHub Disscussions.

**Reply by parasti** [reply]
Good to see you here!
